src/screens/GroupInfo/index.js

```diff
@@ -449,5 +453,18 @@ export const styles = StyleSheet.create({
   },
   scrollContainer: {
     paddingBottom: 30
+  },
+  modalActUser: {
+    padding: 16,
+    paddingLeft: 27,
+    paddingRight: 70
+  },
+  textActUser: {
+    fontFamily: 'Inter',
+    fontStyle: 'normal',
+    fontWeight: '500',
+    fontSize: 18,
+    color: COLORS.black000,
+    paddingVertical: 20
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles `modalActUser` and `textActUser` have been added. Make sure these styles are being used in the component, otherwise, they will just add unnecessary lines of code.
```diff
+  modalActUser: {
+    padding: 16,
+    paddingLeft: 27,
+    paddingRight: 70
+  },
+  textActUser: {
+    fontFamily: 'Inter',
+    fontStyle: 'normal',
+    fontWeight: '500',
+    fontSize: 18,
+    color: COLORS.black000,
+    paddingVertical: 20
+  }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->